chain-storage-sqlite-old/src/lib.rs

```diff
@@ -15,6 +14,14 @@ where
     dummy: std::marker::PhantomData<B>,
 }
 
+pub struct SQLiteBlockStoreConnection<B>
+where
+    B: Block,
```

Comment:
do we need this `Block` trait constraint?

Reply:
see https://github.com/input-output-hk/chain-libs/pull/240#discussion_r365939036

Reply:
Well, now I see that we can actually remove the constraint from the structure and put constraints on the functions which actually use the `Block` trait.

Reply:
It may be better for consistency that a connection works with the same block type. Then again, the same can be said about `SQLiteBlockStore`...